roles/epel_repositories/tasks/main.yml

```diff
@@ -2,6 +2,7 @@
 - name: 'Setup Epel Repository'
   yum:
     name: "{{ epel_repositories_name }}"
+    disable_gpg_check: True
```

Comment:
AFAIK this isn't always needed because on CentOS we typically install `epel-release` from CentOS extras which is GPG signed. Perhaps we need to split this into a RHEL and CentOS task?

Reply:
No, we always install from URL:
https://github.com/theforeman/forklift/blob/8139cc3eaf2323013c0073e331ad69b482425cf1/roles/epel_repositories/defaults/main.yml#L4

Reply:
(unless the user overrides this to `epel-release`, but ain't nobody does that)

Reply:
hmm ok